app/api/api_v1/routers/search.py

```diff
@@ -76,7 +76,7 @@ def _search_request(
                 data_access_search_response,
                 limit=search_body.limit,
                 offset=search_body.offset,
-            )
+            ).increment_pages()
```

Comment:
I would argue that calling `process_vespa_search_reposonse` should also handle the page numbering. We are in a router here and this should not contain any search response logic. Having said that there are functions that exist at the bottom of the file with vespa logic in :cry:  -  probably a tech debt ticket